app/views/layouts/_nav.html.haml

```diff
@@ -2,7 +2,7 @@
 - if simplified_nav
   - pages = ['Home']
 - else
-  - pages = ['About', 'Schedule', 'Travel', 'FAQ', 'Sponsors']
+  - pages = ['Timer', 'Announcements', 'Important Info', 'Schedule', 'Challenges', 'Slack']
```

Comment:
With the changes to the pages, it is going to change the home page. We do not want something like that, but simply need to do an if statement to check what is calling this file

Reply:
I just went ahead and made a separate nav layout because I figured we were going to add the logo to the new nav anyways

Reply:
But with the changes the new nav is going to overwrite the index, and with that might cause issues on navigating between pages, as well as going to specific sections.